include/envoy/event/scaled_timer_minimum.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <chrono>
+#include <ostream>
```

Comment:
I'm worried that adding this and the related inline methods here may have a detrimental effect on compile-time. Does this need to live in a header?

Reply:
The method definitions could be moved to a .cc file. Should I put that under `include` or move the whole thing to `source/common/common`?

Reply:
These ostream helpers don't have to be defined inside the struct and you don't even need `friend` since the members are public, so we can define in another .h files, where are they used?